.github/workflows/release.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       - uses: goreleaser/goreleaser-action@9ed2f89a662bf1735a48bc8557fd212fa902bebf # v6.1.0
         with:
           version: '~> v2'
-          args: release --clean
+          args: release --clean --parelelism=1
```

Comment:
can you add a comment why restrict parallelism to 1

Reply:
done